src/open_prime_hunters_rando/string_tables/string_tables.py

```diff
@@ -0,0 +1,192 @@
+import copy
+import enum
+import typing
+from typing import Any, Self
+
+import construct
+from construct import (
+    Aligned,
+    Array,
+    Byte,
+    Container,
+    CString,
+    If,
+    Int16ul,
+    Int32ul,
+    ListContainer,
+    PaddedString,
+    Pointer,
+    Struct,
+    this,
+)
+
+from open_prime_hunters_rando.constants import EnumAdapter
+
+
+class ScanCategory(enum.Enum):
+    NONE = 0
+    BIOFORM = 0x42
+    EQUIPMENT = 0x45
+    LORE = 0x4C
+    OBJECT = 0x4F
+    OBJECT2 = 0x58
+    OBJECT3 = 0x6F
+    SWITCH = 0x78
+
+
+ScanIconConstruct = EnumAdapter(ScanCategory, Byte)
+
+StringEntryHeader = Struct(
+    "string_id" / PaddedString(4, "ascii"),
+    "_data_offset" / Int32ul,
+    "_string_length" / Int16ul,
+    "scan_speed" / Byte,
```

Comment:
this might be an enum too. what values are used in the vanilla game?

Reply:
1, 2, 3. Good catch